pkg/certificate/providers/vault/certificate_manager.go

```diff
@@ -23,7 +23,7 @@ const (
 	issuingCAField   = "issuing_ca"
 
 	checkCertificateExpirationInterval = 5 * time.Second
-	tmpCertValidityPeriod              = 1 * time.Second
+	decade                             = 8765 * time.Hour
```

Comment:
wait, not counting leap years?
unacceptable

Reply:
:)